app/assets/javascripts/listings/templates/listing/_panel-apply.html.slim

```diff
@@ -9,40 +9,91 @@
       ' {{application.applicationSubmittedDate | date : "MMM d, yyyy" }}
 
 .content-column ng-if="!submittedApplication()"
-  .content-group.has-divider ng-if="listing.Accepting_Online_Applications"
-    h2.h-caps-lined
-      | How to Apply
-    button.button.primary.expand.no-margin type="button" data-event="gtm-apply" title="{{'LABEL.APPLY_ONLINE' | translate}}" ui-sref="dahlia.short-form-welcome.intro({id: listing.Id})" ng-if="!hasDraftApplication()"
-      | {{'LABEL.APPLY_ONLINE' | translate}}
-    .print-only.show-for-print
-      | at housing.sfgov.org
-    button.button.primary.expand.no-margin type="button" data-event="gtm-continue-application" title="{{'LABEL.CONTINUE_APPLICATION' | translate}}" ui-sref="dahlia.short-form-application.name({id: listing.Id})" ng-if="hasDraftApplication()"
-      | {{'LABEL.CONTINUE_APPLICATION' | translate}}
-    .option-divider
-      span.or
-        | or
-  .content-group.has-divider
-    h2.h-caps-lined ng-if="!listing.Accepting_Online_Applications"
-      | How to Apply
-    h3.content-group_title
-      | Pickup and turn in application
-    p.content-group_address.no-margin
-      | {{listing.Application_Organization}}
-    p.content-group_address
-      | {{ listing.Application_Street_Address }},
-      br
-      | {{ listing.Application_City }} {{ listing.Application_State }} {{ listing.Application_Postal_Code }}
-    a.t-small.d-block.ui-link.margin-bottom.hide-for-print title="View Map" target="_blank" ng-href="https://www.google.com/maps/place/{{formattedApplicationAddress(listing)}}"
-      | View Map
-    .content-group_hours ng-if="listing.Office_Hours"
+
+  span ng-if="listingIs('280 Brighton')"
```

Comment:
markup is pretty different so just took the pattern and encapsulated it here as `span ng-if="listingIs('280 Brighton')"`